README.md

```diff
@@ -1,132 +1,109 @@
 <img align="left" width="440" height="140" alt="Fastest full PostgreSQL nodejs client" src="https://raw.githubusercontent.com/porsager/postgres/master/postgresjs.svg?sanitize=true" />
 
-- [🚀 Fastest full featured PostgreSQL node client](https://github.com/porsager/postgres-benchmarks#results)
+- [🚀 Fastest full-featured PostgreSQL node client](https://github.com/porsager/postgres-benchmarks#results)
 - 🚯 1850 LOC - 0 dependencies
 - 🏷 ES6 Tagged Template Strings at the core
 - 🏄‍♀️ Simple surface API
+- 🖊️ Dynamic query support
 - 💬 Chat on [Gitter](https://gitter.im/porsager/postgres)
 
 <br>
 
+# Table of Contents
+
```

Comment:
Should we perhaps move the table of contents below getting started?

Reply:
Yes, will do